### PR TITLE
Remove 2019.3 support and fix all FIX_WHEN_MIN_IS_201 items

### PR DESCRIPTION
**Pull request by hunterwerlla**
- Remove 2019.3 profile
- Remove 2019.3 run configs
- Make default profile 2020.1
- Move src/test 201+ stuff to src/tst
- Fix all `FIX_WHEN_MIN_IS_201` items
- As a bonus the explorer refreshes way faster now
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by abrooksv** [comment]
PR to IJ SDK to add new background pool method: https://github.com/JetBrains/intellij-community/pull/1459

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/editor/S3ViewerPanel.kt [hunk 1]
```diff
@@ -34,7 +35,13 @@ class S3ViewerPanel(disposable: Disposable, private val project: Project, privat
     private val rootNode: S3TreeDirectoryNode = S3TreeDirectoryNode(virtualBucket, null, "")
 
     init {
-        val structureTreeModel: StructureTreeModel<SimpleTreeStructure> = StructureTreeModel(SimpleTreeStructure.Impl(rootNode), disposable)
+        val structureTreeModel: StructureTreeModel<SimpleTreeStructure> = StructureTreeModel(
+            SimpleTreeStructure.Impl(rootNode),
+            null,
+            // TODO this has a concurrency of 1, do we want to adjust this?
+            Invoker.Background(disposable),
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/editor/S3ViewerPanel.kt:42]: If this is a concurrency of 1, use `Invoker.forBackgroundThreadWithoutReadAction`

File: jetbrains-core/tst/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerTreeStructureProviderTest.kt [hunk 2]
```diff
@@ -69,7 +70,7 @@ class AwsExplorerTreeStructureProviderTest {
 
     private fun createTreeModel(): TreeModel {
         val awsTreeModel = AwsExplorerTreeStructure(projectRule.project)
-        val structureTreeModel = StructureTreeModel(awsTreeModel, disposableRule.disposable)
+        val structureTreeModel = StructureTreeModel(awsTreeModel, null, Invoker.Background(disposableRule.disposable), disposableRule.disposable)
```
Comment (abrooksv) [inline jetbrains-core/tst/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerTreeStructureProviderTest.kt:73]: `forBackgroundThreadWithoutReadAction`

File: jetbrains-core/tst/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerNodeProcessorTest.kt [hunk 3]
```diff
@@ -100,7 +101,7 @@ class AwsExplorerNodeProcessorTest {
 
     private fun createTreeModel(): TreeModel {
         val awsTreeModel = AwsExplorerTreeStructure(projectRule.project)
-        val structureTreeModel = StructureTreeModel(awsTreeModel, disposableRule.disposable)
-        return AsyncTreeModel(structureTreeModel, true, disposableRule.disposable)
+        val structureTreeModel = StructureTreeModel(awsTreeModel, null, Invoker.Background(disposableRule.disposable), disposableRule.disposable)
+        return AsyncTreeModel(structureTreeModel, false, disposableRule.disposable)
```
Comment (abrooksv) [inline jetbrains-core/tst/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerNodeProcessorTest.kt:105]: Why are we not showing loading nodes?

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/editor/S3ViewerPanel.kt (hunk 1 above)
Comment (hunterwerlla) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/editor/S3ViewerPanel.kt]: Updated

File: jetbrains-core/tst/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerTreeStructureProviderTest.kt (hunk 2 above)
Comment (hunterwerlla) [reply jetbrains-core/tst/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerTreeStructureProviderTest.kt]: ditto

**Review by abrooksv — APPROVED** [review]

**Pull request merged**